sdk/keyvault/azure-security-keyvault-keys/README.md

```diff
@@ -184,7 +184,7 @@ KeyVaultKey key = keyClient.getKey("<key-name>");
 // Update the expiry time of the key.
 key.getProperties().setExpiresOn(OffsetDateTime.now().plusDays(30));
 KeyVaultKey updatedKey = keyClient.updateKeyProperties(key.getProperties());
-System.out.printf("Key's updated expiry time: %s\n", updatedKey.getProperties().getExpiresOn().toString());
+System.out.println("Key's updated expiry time: " + updatedKey.getProperties().getExpiresOn().toString());
```

Comment:
do we need the explicit toString()?

Reply:
Good catch, we shouldn't need it as far as I know.